Add settings to documentation (#11)
fytinnovations/oc-helpdesk: getting-started/configuration.md

# Configuration

The plugin contains some configuration settings which you may find useful. If you wish to request some additional settings you can put a [feature request](https://github.com/fytinnovations/oc-helpdesk/issues).

## Notifications

> The default mode for notifications is email. Please ensure that you have a working SMTP Server configured before enabling the below features.

Reviewer: ```suggestion
The default mode for notifications is email. Please ensure that you have a working SMTP Server configured before enabling the below notifications.
```

> ### Notify agents when tickets are assigned to them

Reviewer: ```suggestion
### Notify all the assignee when a ticket is assigned.
```

> ### Notify agents when tickets are assigned to them

Reviewer: ```suggestion
### Notify all the assignee when a ticket is assigned
```

> Enable this feature will notify the agent via email when a ticket is assigned to them by the supervisor.
> 
> ### Notify supervisors when new tickets are created
> 
> This feature will email supervisors when there is a ticket submitted by the end user.
> 
> ### Notify users and agents when there is a comment on the ticket
> 
> After the ticket has been created and assigned to a agent. There can be a conversation on the basis of the ticket. Enabling this feature will email the agent and the user on each and every comment.
> 
> ### Notify users, supervisor, assignees when there is a change in the status of the ticket
> 
> When there is a change in the status of the ticket "i.e the ticket get closed" the respective users will be notified.
> 
> ### Notify agents when there is a private comment on the ticket
> 
> This plugin provides a feature for posting private comments so that different agents can talk to each other without the knowledge of the end user. Enabling this feature will notify the agents for those private comments.

Reviewer: ```suggestion
Support agents will be notified when a ticket is assigned to them by the supervisor

### Notify supervisors when new a ticket is created

Supervisors will be notified whenever a new ticket is submitted by the end user.

### Notify users and all the assignee when there is a comment on the ticket

All the assignee and the end user be notified for every public comment on the ticket

### Notify users, supervisor and all the assignee when there is a change in the status of the ticket

When the status of the ticket changes the end user, supervisor and all the assignee will be notified

### Notify all the assignee when there is a private comment on the ticket

This plugin provides a feature for posting private comments so that different agents can talk to each other without the knowledge of the end user. Enabling this feature will notify the agents for those private comments.
```
